### PR TITLE
adding it

### LINKED ISSUE #1383: sql-lint configuration file support
**Is your feature request related to a problem? Please describe.**

`sql-lint` has configuration file support but it's not present in super-linter documentation, and not able to be specified in an environment variable.

**Describe the solution you'd like**

JSON-based configuration files are already supported: https://sql-lint.readthedocs.io/en/latest/files/configuration.html

This should work fine with the super-linter with defaults, but you're awesome and like override support, so add a way to set an environment variable, document it, and send that into the invocation like you do with other linters.

**Describe alternatives you've considered**

As stated this works as-is, as long as you don't need to override the configuration file name.

### PR DESCRIPTION
**Pull request by admiralAwkbar**
This closes #1383 

This will now allow `SQL-Lint` to use a configuration file if needed.

This seems to have been halfway implemented but needed to be closed out :)

### CODE REVIEW

**Comment by admiralAwkbar** [comment]
@withinfocus 
Any chance you can test this?

File: lib/linter.sh [hunk 1]
```diff
@@ -147,7 +147,7 @@ SNAKEMAKE_SNAKEFMT_FILE_NAME="${SNAKEMAKE_SNAKEFMT_CONFIG_FILE:-.snakefmt.toml}"
 # shellcheck disable=SC2034  # Variable is referenced indirectly
 SUPPRESS_POSSUM="${SUPPRESS_POSSUM:-false}"
 # shellcheck disable=SC2034  # Variable is referenced indirectly
-SQL_FILE_NAME=".sql-config.json"
+SQL_FILE_NAME="${SQL_CONFIG_FILE}:-.sql-config.json"
```
Comment (withinfocus) [inline lib/linter.sh:150]: The brace needs to be shifted to the right before the end quote.

**Review by withinfocus — CHANGES_REQUESTED** [review]

**Comment by withinfocus** [comment]
@admiralAwkbar I can likely get a local image going -- minor brace position change needed first though.

**Comment by admiralAwkbar** [comment]
@withinfocus seems i cant read ;)

**Review by withinfocus — APPROVED** [review]
This checks out from my chair, although I wasn't able to pull your container image for this PR (broken Go setup?). The inferred invocation e.g.:

> npx sql-lint {file} --config .sql-config.json

is working as expected and reading the local config.

**Pull request merged**